File: VL17.py
import math

def count_divisors(number:int):
    if number < 0:
        return 0
    count  = 0
    for i in range(1,int(math.sqrt(number))+1):
        if number % i == 0:
            if i * i == number:
                count  += 1
            else:
                count  += 2
    return count

print(count_divisors(abs(int(input()))))

File: VL17/VL17.cpp
#include<bits/stdc++.h>
using namespace std;

int count_divisors(int number) {
    if (number < 0) return 0;
    int count = 0;
    for (int i = 1; i <= sqrt(number); i++) {
        if (number % i == 0) {
            if (i * i == number) {
                count += 1;
            } else {
                count += 2;
            }
        }
    }
    return count;
}

int main() {
    int n;
    cin >> n;
    cout << count_divisors(abs(n)) << endl;
    return 0;
}
